webapp/Connector/src/view/MabGrid.js

```diff
@@ -161,107 +156,145 @@ Ext.define('Connector.view.MabGrid', {
         return this.exportExcelButton;
     },
 
-    getRReportButtons: function() {
-        var reports = [];
-        for (var i = 1 ; i < 3; i++) {
-            var reportId = LABKEY.getModuleProperty('cds', Connector.view.MabGrid.MAbReportID_PROP_PREFIX + i);
-            var reportLabel = LABKEY.getModuleProperty('cds', Connector.view.MabGrid.MAbReportLabel_PROP_PREFIX + i);
-            if (reportId && reportLabel) {
-                reports.push(Ext.create('Ext.button.Button', {
-                    cls: 'mabgridcolumnsbtn',
-                    text: reportLabel,
-                    handler: (function(id, label){
-                        return function(){
-                            this.showRReport(id, label);
+    getReportButtons : function() {
+        if (this.reportButtons === undefined) {
+            var items = [];
+            this.reportButtons = [];
+
+            for (var i = 1; i < 3; i++) {
+                var reportId = LABKEY.getModuleProperty('cds', Connector.view.MabGrid.MAbReportID_PROP_PREFIX + i);
+                var reportLabel = LABKEY.getModuleProperty('cds', Connector.view.MabGrid.MAbReportLabel_PROP_PREFIX + i);
+
+                if (reportId && reportLabel) {
+                    items.push({
+                        xtype: 'button',
+                        cls: 'mabgridcolumnsbtn',
+                        handler: (function(id, label) {
+                            return function() {
+                                this.showRReport(id, label);
+                            }
+                        })(reportId, reportLabel),
+                        text: reportLabel,
+                        scope: this
+                    });
+                }
+            }
+
+            if (items.length) {
+                this.reportButtons.push({
+                    xtype: 'container',
+                    disabled: !this.allowReporting,
+                    disabledCls: 'vardisable',
+                    maskOnDisable: false,
+                    items: items,
+                    listeners: {
+                        afterrender: {
+                            fn: function(ct) {
+                                this.reportButtonContainer = ct;
+
+                                ct.getEl().on({
+                                    mouseenter: function() {
+                                        if (!this.allowReporting) {
+                                            ChartUtils.showCallout({
+                                                target: ct.getEl().dom,
+                                                placement: 'bottom',
+                                                title: 'Enable reporting',
+                                                xOffset: -65,
```

Comment:
Production site currently have 2 reports. If the tooltip binding is not per report, consider adjusting offset so that arrow is somewhere in between the 2 buttons. Right now when you hover the 2nd report, the popup arrow points at the first report.

Reply:
Adjusted the arrow to be towards the middle of the report buttons.